gcp/gcs.tf

```diff
@@ -0,0 +1,27 @@
+resource "random_id" "bucket_suffix" {
+  byte_length = 12
+}
+
+resource "google_storage_bucket" "nomad" {
+  location                    = "US"
```

Comment:
It's probably fine, but just double-checking, does this value need to be adjusted if the user picks a non-US region and zone?

Reply:
Hm, good question. The GCS location is required, but there's no way to map from GCE regions to GCS locations. I tested with `location="EU"` and everything worked fine though, so I think leaving this hardcoded to US is the best solution.